Rubberduck.Inspections/Concrete/ImplicitByRefModifierInspection.cs

```diff
@@ -0,0 +1,74 @@
+using System.Collections.Generic;
+using System.Linq;
+using Antlr4.Runtime;
+using Rubberduck.Common;
+using Rubberduck.Inspections.Abstract;
+using Rubberduck.Inspections.Results;
+using Rubberduck.Parsing;
+using Rubberduck.Parsing.Grammar;
+using Rubberduck.Parsing.Inspections.Abstract;
+using Rubberduck.Parsing.Inspections.Resources;
+using Rubberduck.Parsing.VBA;
+using Rubberduck.VBEditor;
+
+namespace Rubberduck.Inspections.Concrete
+{
+    public sealed class ImplicitByRefModifierInspection : ParseTreeInspectionBase
+    {
+        public ImplicitByRefModifierInspection(RubberduckParserState state)
+            : base(state, CodeInspectionSeverity.Hint)
+        {
+        }
+
+        public override CodeInspectionType InspectionType => CodeInspectionType.CodeQualityIssues;
+
+        public override IInspectionListener Listener { get; } = new ImplicitByRefModifierListener();
```

Comment:
Does this create a new instance everytime the getter is called? If so then that would be a bug..